mithril-signer/src/aggregator_client.rs

```diff
@@ -47,61 +47,90 @@ pub enum CertificateHandlerError {
     /// Incompatible API version error
     #[error("HTTP API version mismatch: {0}")]
     ApiVersionMismatch(String),
+
+    /// Client creation error
+    #[error("client creation failed: {0}")]
+    ClientCreation(String),
```

Comment:
Could you rename it to `HttpClientCreation` else I don't really know what this 'client' is aboult